### ISSUE TITLE
Updating to Sequoia

### ISSUE THREAD

**Issue by Blulyk** [shammu2bd/Asus-X540LJ-Hackintosh-macOS-Sequoia-OpenCore#2]
Updating the system to sequoia, works as fine as Sonoma following the same method?

**Comment by shammu2bd** [comment]
yes pretty much same. but i have updated my efi to 0.9.8 and some kext. then updated to macos 15. now woroking very good.

**Comment by shammu2bd** [comment]
also update the OCLP patching to leatest version

**Comment by Blulyk** [comment]
Nice!
Do you have plans of uploading the files?

**Comment by Blulyk** [comment]
Man?

**Comment by shammu2bd** [comment]
sorry for late reply. I have been busy on my project. Yes i will update the EFI. wait for some days.

**Comment by Blulyk** [comment]
Ok, one last question, what wifi and bluetooth adapter do you use?

**Comment by shammu2bd** [comment]
right now inbuilt wifi is working fine. but bluetooth still broked. WIFI: Atheros 9565

**Comment by Blulyk** [comment]
How did you made inbuilt wifi work, man this seems like magic for me

**Comment by shammu2bd** [comment]
Opencore legecy patcher (OCLP) + extra kext + EFI patching made it worked. After montery update it was totally broked. but recenty i found the fix. not it works montery to sequoia.

**Comment by Blulyk** [comment]
Would you include the fix in the sequoia update?

**Comment by shammu2bd** [comment]
yes. ofcourse

**Comment by shammu2bd** [comment]
@Blulyk hey. Sorry for late. I have updated the EFI file and simplified the installing documentation. Now it supports macOS 15.1 and built-in wifi. try this and let me know if all are works properly.
